crates/core_arch/src/x86/avx512f.rs

```diff
@@ -6863,17 +6863,13 @@ pub unsafe fn _mm_maskz_getmant_pd(
 /// [Intel's documentation](https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm512_add_round_ps&expand=145)
 #[inline]
 #[target_feature(enable = "avx512f")]
-#[cfg_attr(test, assert_instr(vaddps, rounding = 8))]
-#[rustc_args_required_const(2)]
-pub unsafe fn _mm512_add_round_ps(a: __m512, b: __m512, rounding: i32) -> __m512 {
+#[cfg_attr(test, assert_instr(vaddps, IMM4 = 8))]
+#[rustc_legacy_const_generics(2)]
+pub unsafe fn _mm512_add_round_ps<const IMM4: i32>(a: __m512, b: __m512) -> __m512 {
```

Comment:
There's no reason to change the change from `rounding` to `IMM4`. Using `ROUNDING` is better.

Reply:
It might be better to create a specific macro just for this condition since it is used quite often.

Reply:
Yes, you can have multiple constant arguments. You need to use 2 separate `static_assert` for them.